src/Suggestor/Suggestor.ts

```diff
@@ -451,9 +451,9 @@ function addDependsOnSuggestions(
         const newTaskToAppend = dependsOnMatch[3];
 
         // Find all Tasks, Already Added
-        let blockingTasks = [] as Task[];
+        let blockingTasks: Task[] = [];
         if (existingDependsOnIdStrings) {
-            blockingTasks = allTasks.filter((task) => task.id && existingDependsOnIdStrings.contains(task.id));
+            blockingTasks = allTasks.filter((task) => task.id && existingDependsOnIdStrings.includes(task.id));
```

Comment:
Just to note that this change was needed because `string.contains()` is an Obsidian-provided extension to `string` that is not available when the code is run from unit tests.